docker/Dockerfile

```diff
@@ -14,18 +14,18 @@ RUN apt-get update
 COPY ./requirements /requirements
 COPY ./r-requirements /r-requirements
 
-RUN apt-get -y install build-essential libxml2-dev zlib1g-dev python-dev pkg-config libffi6 libffi-dev libcairo-dev libssl-dev libcurl4-openssl-dev \
+RUN apt-get -y install build-essential libxml2-dev zlib1g-dev python-dev pkg-config libffi6 libffi-dev libssl-dev libcurl4-openssl-dev \
                        python3.6-dev python3-pip ipython3 \
                        r-base gfortran \
                        libc6-i386 lib32stdc++6
 
 
 RUN python3.6 -m pip install pip --upgrade
 RUN python3.6 -m pip install wheel
-RUN pip install -r /requirements/${APP_ENV}.txt 
+RUN pip install -r /requirements/${APP_ENV}.txt
 # RUN apt-get -y install libmariadb-client-lgpl-dev libssl-dev libcurl4-openssl-dev
 
-# RUN R --vanilla -q -f /r-requirements/${APP_ENV}.R
+RUN Rscript --vanilla /r-requirements/${APP_ENV}.R
```

Comment:
What is Rscript?

Reply:
It's a convenience wrapper that comes with R to execute an R script from the command line. Basically that's what you put in the shebang line if you want to write standalone R scripts.

Reply:
Nice